iset.mm

```diff
@@ -140477,6 +140525,169 @@ S C_ ( P ( ball ` D ) T ) ) $=
       DAUCUBUABCYHYIAUUFUXCBSYIVFUUKJBESGFYFWSUWJUWKWRYG $.
   $}
 
+  ${
+    $d x B $.  $d x C $.  $d x D $.  $d x H $.  $d x X $.  $d x A $.  $d x Y $.
+    limccnp2.r $e |- ( ( ph /\ x e. A ) -> R e. X ) $.
+    limccnp2.s $e |- ( ( ph /\ x e. A ) -> S e. Y ) $.
+    limccnp2.x $e |- ( ph -> X C_ CC ) $.
+    limccnp2.y $e |- ( ph -> Y C_ CC ) $.
+    limccnp2cntop.k $e |- K = ( MetOpen ` ( abs o. - ) ) $.
+    limccnp2.j $e |- J = ( ( K tX K ) |`t ( X X. Y ) ) $.
+    limccnp2.c $e |- ( ph -> C e. ( ( x e. A |-> R ) limCC B ) ) $.
+    limccnp2.d $e |- ( ph -> D e. ( ( x e. A |-> S ) limCC B ) ) $.
+    limccnp2.h $e |- ( ph -> H e. ( ( J CnP K ) ` <. C , D >. ) ) $.
+
+    ${
+      $d A d $.  $d B d $.  $d C d r s $.  $d D d r s $.  $d E d r s $.
+      $d F d x $.  $d G d x $.  $d H d r s $.  $d L r s $.  $d R d r s $.
+      $d S d s $.  $d X r s $.  $d Y r s $.
+      limccnp2lem.nf $e |- F/ x ph $.
+      limccnp2lem.e $e |- ( ph -> E e. RR+ ) $.
+      limccnp2lem.j $e |- ( ph -> L e. RR+ ) $.
+      limccnp2lem.rs $e |- ( ph -> A. r
+                                     e. X
+                                        A. s
+                                        e. Y
+                                           ( ( ( C
+                                                   ( ( abs o. - )
+                                                   |` ( X X. X ) )
+                                                   r )
+                                                 <
+                                                 L
+                                              /\ ( D
+                                                   ( ( abs o. - )
+                                                   |` ( Y X. Y ) )
+                                                   s )
+                                                 <
+                                                 L )
+                                           -> ( ( C H D )
+                                                ( abs o. - )
+                                                ( r H s ) )
+                                              <
+                                              E ) ) $.
```

Comment:
this sems to be mmj2 style. In set.mm, this hypothesis (and the following ones) should be compressed in fewer lines.

Reply:
Yeah, I'll fix it, just forgot.

Reply:
Now fixed.